src/lib/containers/access_requirement_list/managedACTAccess/RequestDataAccessStep2.tsx

```diff
@@ -450,45 +452,55 @@ const RequestDataAccessStep2: React.FC<RequestDataAccessStep2Props> = props => {
   }
 
   const uploadCallback = (data: UploadCallbackResp) => {
-    if (data.context === 'attachments') {
```

Comment:
This is a terrible diff, I wrapped the existing code with:
if (data.resp && data.success) {
...
}